mirrord/agent/src/steal/connections/filtered.rs

```diff
@@ -554,7 +555,13 @@ where
     }
 
     /// Handles a [`Request`] intercepted by the [`FilteringService`].
-    #[tracing::instrument(level = "trace", skip(self, request, tx), ret, err(Debug))]
+    #[tracing::instrument(
+        level = Level::TRACE,
+        skip(self, tx),
+        fields(?request = request.request),
+        ret,
+        err(level = Level::WARN)
+    )]
```

Comment:
It can return an error on the channels `.send`.

```rust
        if self.subscribed.insert(client_id, true).is_none() {
            // First time this client will receive a request from this connection.
            tx.send(ConnectionMessageOut::SubscribedHttp {
                client_id,
                connection_id: self.connection_id,
            })
            .await?; <---
        }

```

Reply:
But the `ret` is pretty useless, removed it.